### ISSUE TITLE
Add getDate and setDate methods to Messages API

### ISSUE THREAD

**Issue by ghost** [blackberry/WebWorks#11]
The existing blackberry.message.Message API is currently missing a 'date' field.  Recommend adding get/set methods.

https://github.com/blackberry/WebWorks/blob/master/api/messaging/src/main/java/blackberry/messaging/message/MessageObject.java

**Comment by tneil** [comment]
I agree that we should have both the received and sent dates but these should be readonly properties.  These are set by the system when the message is received or sent

I would suggest:

readonly Date receivedDate;
readonly Date sentDate;

Reference:
http://www.blackberry.com/developers/docs/5.0.0api/net/rim/blackberry/api/mail/Message.html
